src/main/scala/org/apache/mesos/chronos/scheduler/jobs/TaskUtils.scala

```diff
@@ -32,6 +32,15 @@ object TaskUtils {
     taskIdPattern.findFirstIn(taskIdString).nonEmpty
   }
 
```

Comment:
RichBoolean is only used in appendSchedulerMessage and is a bit confusing in terms of readability. Could we instead just do the if (taskStatus.hasMessage && taskStatus.getMessage.nonEmpty) Some(taskStatus.getMessage) else None inside the appendSchedulerMessage?